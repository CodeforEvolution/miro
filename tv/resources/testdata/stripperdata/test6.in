<p><a href="blah">abc</a>def</p>
