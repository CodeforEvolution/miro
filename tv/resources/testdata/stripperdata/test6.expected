(u'abcdef', [(0, 3, u'blah')])
